src/main/java/ch/njol/skript/expressions/ExprEnchantItemExpCost.java

```diff
@@ -0,0 +1,127 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import org.bukkit.event.Event;
+import org.bukkit.event.enchantment.EnchantItemEvent;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.ScriptLoader;
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Events;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.log.ErrorQuality;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("Enchant Event Experience Cost")
+@Description({"The cost to enchant in an enchant event.", 
+				"This is the number displayed in the enchantment table, not the actual number of levels removed."})
+@Examples({"on enchant:",
+			"\tsend \"Cost: %cost of enchanting%\" to player"})
+@Events("enchant prepare")
+@Since("INSERT VERSION")
+public class ExprEnchantItemExpCost extends SimpleExpression<Number> {
+
+	static {
+		Skript.registerExpression(ExprEnchantItemExpCost.class, Number.class, ExpressionType.SIMPLE, 
+				"[the] [e]xp[erience] cost (of|for) [the] enchant[ment]",
```

Comment:
i think it would make more sense to add an enchantment event value to the event and just piggy back off the cost property expression that already exists. it would allow for the same syntax is less code

Reply:
So, this expression is actually the cost that was displayed on the side of the enchantment offer. From what I saw in testing, it doesn't actually do anything if you modify it (I'll test again though), so I'm not why it's even modifiable in the first place. However, since the code is pretty much already there, maybe someone will find a use for being able to modify it. I'll clean up the  code & docs on this one.